src/odemis/acq/test/stream_test.py

```diff
@@ -2964,6 +2967,114 @@ def test_streak_acq_integrated_images_leech(self):
         temporalSpectrum_drift = ts_da[-1]  # drift correction image
         self.assertGreaterEqual(temporalSpectrum_drift.shape[-4], 2)
 
+    def test_acq_cl_se_ebic(self):
+        """
+        Test short & long acquisition for SE CL and EBIC acquisition simultaneously
+        """
+        # create axes
+        axes = {"filter": ("band", self.filter)}
+
+        # Create the stream
+        sems = stream.SEMStream("test sem", self.sed, self.sed.data, self.ebeam,
+                                emtvas={"dwellTime", "scale", "magnification", "pixelSize"})
+        mcs = stream.CLSettingsStream("test",
+                                      self.cl, self.cl.data, self.ebeam,
+                                      axis_map=axes,
+                                      emtvas={"dwellTime", })
+        mcs_two = stream.CLSettingsStream("testTWO",
```

Comment:
Let's change it to a EBICSettingsStream. Mainly to use that new class, and also because it feels more logical in this test case.